src/ResourceManager/MachineLearning/Commands.MachineLearning.Test/TestData/GraphWebServiceDefinition_Dogfood.json

```diff
@@ -0,0 +1,190 @@
+﻿{
+  "properties": {
+    "title": "ServiceTitle",
+    "description": "This is some service description",
+    "readOnly": "false",
+    "keys": {
+      "primary": "S8ayf24PAsbQqy8Nm67uGf3MbOeorYKPY8QfXen2+JsBZV5Q0NFfk9tgC7mjJ3aCsUPY8yKCFVcI/sCcOlmkKw==",
```

Comment:
It is the key to access the web service you created with the cmdlet. It is part of the data structure here.